spec/integration/core/vm_spec.rb

```diff
@@ -276,7 +278,7 @@
           pending("There is only one StorageProfile in vDC #{@test_params.vdc_1_name}: cannot test.")
         end
         expect{ @vm.update_storage_profile(@test_params.storage_profile) }.
-          to raise_error(Fog::Compute::VcloudDirector::TaskError)
+          to raise_exception(Fog::Compute::VcloudDirector::TaskError, /attached to a virtual machine/)
```

Comment:
Don't know why I did that actually, `raise_error` is an alias to `raise_exception` so they are both functionally the same. Happy to change back.